tests/commands/run_test.py

```diff
@@ -634,6 +642,13 @@ def test_skip_bypasses_installation(cap_out, store, repo_with_passing_hook):
     )
     assert ret == 0
 
+    ret, printed = _do_run(
+        cap_out, store, repo_with_passing_hook,
+        run_opts(all_files=True),
+        {'SKIP': 'skipme-1'},
+    )
+    assert ret == 1
```

Comment:
this test now validates two separate things -- it should be two separate tests

Reply:
> just future open source advice, don't make PRs from your `main` branch as it makes it significantly more difficult to review / fix

Sure. I was in the gerrit workflow mindset.

I can put it in a separate test method.

I see the CI failing, should I rebase on `v2.20.0` or is this fine?